### ISSUE TITLE
HLint 3.3.5 lacks bindist downloads

### ISSUE THREAD

**Issue by RyanGlScott** [ndmitchell/hlint#1338]
The [HLint 3.3.5 release](https://github.com/ndmitchell/hlint/releases/tag/v3.3.5) appears to lack bindists available for download. Compare this to the [3.3.4 release](https://github.com/ndmitchell/hlint/releases/tag/v3.3.4), which has bindists for Windows, macOS, and Linux. Is this intentional?

**Comment by ndmitchell** [comment]
Not intentional, but I know why it happened (wouldn't build with GHC 9.2 which is what the bindist was set to). The main reason for 3.3.5 was to have aeson-2.0 compatibility, but that's not really necessary for people using bindist. Any disadvantages to just skipping that one?

**Comment by RyanGlScott** [comment]
I opened this issue mostly for the benefit of [`haskell-ci`](https://github.com/haskell-CI/haskell-ci/issues/568), whose `--hlint` mode will attempt to download a bindist for the latest HLint release on Hackage. It's not an insurmountable obstacle by any means—see https://github.com/haskell-CI/haskell-ci/issues/568#issuecomment-998750916 for workarounds—but I did think it worth raising an issue.

If you're inclined to skip a bindist for the 3.3.5 release, I'm fine with that too. Feel free to close this issue if that's the case.
